pontoon/administration/forms.py

```diff
@@ -11,6 +11,10 @@ class Meta:
                   'info_brief', 'url', 'width',
                   'links', 'disabled')
 
+    def __init__(self, *args, **kwargs):
+        super(ProjectForm, self).__init__(*args, **kwargs)
+        self.fields['locales'].required = False
+
```

Comment:
This makes locales optional, which is not what we want.

The problem started after we upgraded Django to 1.10, which apparently adds required attributes to required fields, which forces validation by the browser. Validation naturally happens before the javascript executes, which populates the locales field. So we need to do something like this:

```
// Submit form with button
$('.save').click(function(e) {
  e.preventDefault();
  $('#admin-form').submit();
});
```

Also, we should add `{{ repo_form.url.errors }}` under `{{ repo_form.url }}` to show errors for the repository URL fields.